test.js

```diff
@@ -0,0 +1,27 @@
+const forEmitOf = require("./dist");
```

Comment:
This is handy! Could it go in the examples folder, just to keep things inline?

Reply:
I created some equivalent test cases. This file I uploaded by mistake this time